distribution/docs/src/main/jdocs/content/_plugins/tags-filter-plugin-contents.adoc

```diff
@@ -0,0 +1,23 @@
+:type: plugin
+:status: published
+:title: Tags Filter Plugin
+:link: _tags_filter_plugin
+:plugintypes: prefederatedquery
+:summary: Updates queries without filters.
+
+The Tags Filter Plugin updates queries without filters for tags, and adds a default tag of `resource`.
+For backwards compatibility, a filter will also be added to include metacards without any tags attribute.
+
+===== Related Components to Tags Filter Plugin
```

Comment:
❓ Just noticed that this section isn't always there. Do we want to skip it when there's nothing related to the plugin, or should we always have the section and _None._ in the description? I don't have a strong preference as long as we're consistent.